string node_name
string name
---
bool exists